canary/package.json

```diff
@@ -30,6 +33,7 @@
   "author": "",
   "license": "Apache-2.0",
   "devDependencies": {
+    "globby": "^13.1.1",
```

Comment:
Isn't this already a dep in canaries/scripts?

Reply:
Handled in #1744 -- [287b3e7](https://github.com/aws-amplify/amplify-ui/pull/1744/commits/287b3e7fa3f37d6011b43e564cdad80dddcab219)